src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h

```diff
@@ -228,7 +229,7 @@ namespace OpenMS
 
     /// main method of AccurateMassSearchEngine
     /// input map is not const, since it will get annotated with results
-    void run(FeatureMap&, MzTab&) const;
+    void run(FeatureMap&, MzTab&, MzTabM&) const;
```

Comment:
it seems a bit wasteful to generate both files. especially for accuratemasssearch these could get pretty large.
could this be easily refactored to produce two run methods with minimal code dublication?

Reply:
I did it as it is since I was thinking that it is not easily possible to provide two methods with minima code duplication. But we can take a look together if you like.

Reply:
yeah that would be great.